lib/rubocop/cop/layout/align_hash.rb

```diff
@@ -289,11 +321,13 @@ def correct_key_value(key_delta, key, value, separator)
         end
 
         def new_alignment(key)
-          case cop_config[key]
-          when 'key'       then KeyAlignment.new
-          when 'table'     then TableAlignment.new
-          when 'separator' then SeparatorAlignment.new
-          else raise "Unknown #{key}: #{cop_config[key]}"
+          {
+            'key' => [KeyAlignment.new],
+            'table' => [TableAlignment.new],
+            'separator' => [SeparatorAlignment.new],
+            'table_or_key' => [TableAlignment.new, KeyAlignment.new]
```

Comment:
shoudl I split the string by `or`and use both sides of the or. So you can do either `table_or_key` or `key_or_table`, if it shoudl autofix to the one spesified first we might want both?